packages/zoe/src/contracts/multipoolAutoswap/pool.js

```diff
@@ -215,9 +235,38 @@ export const makeAddPool = (zcf, isSecondary, initPool, centralBrand) => {
         );
 
         userSeat.exit();
+        updateState(pool);
         return 'Liquidity successfully removed.';
       },
+      getNotifier: () => notifier,
+      updateState: () => updateState(pool),
+      // eslint-disable-next-line no-use-before-define
+      getToCentralPriceAuthority: () => toCentralPriceAuthority,
+      // eslint-disable-next-line no-use-before-define
+      getFromCentralPriceAuthority: () => fromCentralPriceAuthority,
     };
+
+    // TODO: if the pool exists, but its liquidity is zero,
+    //  getPriceGivenAvailableInput() throws.
+    const toCentralPriceAuthority = makePriceAuthority(
+      pool.getPriceGivenAvailableInput,
+      secondaryBrand,
+      centralBrand,
+      timer,
+      zcf,
+      notifier,
+      quoteIssuerKit,
+    );
+    const fromCentralPriceAuthority = makePriceAuthority(
+      pool.getPriceGivenAvailableInput,
```

Comment:
Out of ignorance, should both of these `makePriceAuthority` calls' first arguments (`pool.getPriceGivenAvailableInput`) be the same?